extensions/amp-consent/0.1/consent-info.js

```diff
@@ -429,3 +448,26 @@ export function assertMetadataValues(metadata) {
     );
   }
 }
+
+/**
+ * Validates the type of the `exposes` field,
+ * as well as the values.
+ * @param {!JsonObject} config
+ * @return {!Array<EXPOSED_CONSENT_API>}
+ */
+export function validateExposesApi(config) {
+  const exposesApis = [];
+  if (config['exposes']) {
+    const apiList = config['exposes'];
+    userAssert(isArray(apiList), `${TAG}: "exposes" expects array`);
+    for (let i = 0; i < apiList.length; i++) {
+      switch (apiList[i]) {
+        case EXPOSED_CONSENT_API.TCF_POST_MESSAGE:
```

Comment:
should the AMP provided APIs be optional and defined in `exposes` as well?

Reply:
Actually for now, since we are thinking about performance, I'm ok with turning this into `exposesTcfApi: true` instead of `exposes: []`. This will mean less overall code and we can easily expand in the future. WDYT?

Reply:
My hope is that we won't expand this in the future. If we have it, it should be more a scalable solution. That the API should not be implemented by the AMP team, maybe in the format of a plugin.